source/develop/release-management/features/virt/clone-vm.html.md

```diff
@@ -45,7 +45,7 @@ As shown in the following picture a new action button called "Clone VM" has been
 
 When clicked, a new window will be opened containing only the new name of the VM:
 
-![](CloneVmWindow.png‎ "CloneVmWindow.png‎")
+![](CloneVmWindow.png "CloneVmWindow.png")
```

Comment:
When the quotes contains the filename, then I guess we could automate easily a removal. Probably also some cases with general junk into it. Would need to list cases first. The rest would need human supervision I guess.